jooby/src/main/java/org/jooby/WebSocket.java

```diff
@@ -734,4 +734,56 @@ default void send(final Object data, final OnError err) throws Exception {
    */
   void send(Object data, SuccessCallback success, OnError err) throws Exception;
 
+  /**
+   * Send data to all connected sessions.
+   *
+   * If the web socket is closed this method throw an {@link Err} with {@link #NORMAL} close status.
+   *
+   * @param data Data to send.
+   * @throws Exception If something goes wrong.
+   */
+  default void broadcast(final Object data) throws Exception {
+    broadcast(data, SUCCESS, ERR);
+  }
+
+  /**
+   * Send data to all connected sessions.
+   *
+   * If the web socket is closed this method throw an {@link Err} with {@link #NORMAL} close status.
+   *
+   * @param data Data to send.
+   * @param success A success callback.
+   * @throws Exception If something goes wrong.
+   */
+  default void broadcast(final Object data, final SuccessCallback success) throws Exception {
+    broadcast(data, success, ERR);
+  }
+
+  /**
+   * Send data to all connected sessions.
+   *
+   * If the web socket is closed this method throw an {@link Err} with {@link #NORMAL} close status.
+   *
+   * @param data Data to send.
+   * @param err An err callback.
+   * @throws Exception If something goes wrong.
+   */
+  default void broadcast(final Object data, final OnError err) throws Exception {
+    broadcast(data, SUCCESS, err);
+    ;
+  }
+
+  /**
+   * Send data to all connected sessions.
+   *
+   * If the web socket is closed this method throw an {@link Err} with {@link #NORMAL} close status.
+   *
+   * @param data Data to send.
+   * @param success A success callback.
+   * @param err An err callback.
+   * @throws Exception If something goes wrong.
+   */
+  void broadcast(final Object data, final SuccessCallback success, final OnError err)
```

Comment:
Final on interface method that are not default makes no sense since they do not affect the actual implementation.